staging/src/k8s.io/client-go/tools/record/event_test.go

```diff
@@ -631,15 +641,18 @@ func TestMultiSinkCache(t *testing.T) {
 		},
 	}
 	testPod2 := &v1.Pod{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
Why add this? If it's because you want to pass nil for a scheme, maybe think about making/finding a scheme instead... I'll be more comfortable with minimal changes to test code in a change like this--not sure if this change to the test leaves it sufficiently similar to actual uses to remain useful...